### ISSUE TITLE
Installation Error of Aborted

### ISSUE THREAD

**Issue by mfp0610** [Jittor/jittor#341]
## Describe the bug
 
Bug of core dumped when installing Jittor

## Full Log

Using g++ for c/c++ compiler, and the cc_path and nvcc_path have been checked correct: 

![屏幕截图 2022-06-17 164722](https://user-images.githubusercontent.com/56423806/174263411-47165cc8-d1fe-4249-b13d-a088b5e52fbc.png)

**Comment by mfp0610** [comment]
g++ 6.5.0
cuda 10.1

**Comment by qcloudq** [comment]
same. have you solved that?

**Comment by cxchhh** [comment]
Hi there, I encounter the same problem. Have you fix it or locate the error?

**Comment by liang-zijian** [comment]
I encountered the same issue, and this worked for me: let Jittor install CUDA for you instead of using your own CUDA.
try:
`python -m jittor_utils.install_cuda`

**Comment by cxchhh** [comment]
> I encountered the same issue, and this worked for me: let Jittor install CUDA for you instead of using your own CUDA. try: `python -m jittor_utils.install_cuda`

yep, I figured that out too. Guess that's the only way to fix it...
